packages/haste-task-eslint/test/index.spec.js

```diff
@@ -48,4 +48,21 @@ describe('haste-eslint', () => {
 
     expect(test.files['fixable.js'].content).toBe('/foo {2}bar/');
   });
+
+  it('should support formatter', async () => {
```

Comment:
> should support a custom formatter option

What do you say?

Reply:
Actually it's for both built-in formatters and 3rd parties, does is considered custom?

Reply:
good point.

> should support a formatter option

?